lib/src/view/analysis/analysis_screen.dart

```diff
@@ -506,6 +506,9 @@ class _BoardState extends ConsumerState<_Board> {
         showLastMove: boardPrefs.boardHighlights,
         enableCoordinates: boardPrefs.coordinates,
         animationDuration: boardPrefs.pieceAnimationDuration,
+        dragFeedbackSize: boardPrefs.magnifyDraggedPiece ? 2.0 : 1.0,
```

Comment:
Note: This logic is already duplicated (here and in the normal game screen), and once we have the board editor, we'll duplicate it again. Would it make sense to add a method to `BoardPreferences` that converts `BoardPreferences` to `chessground.ChessBoardSettings` and/or `chessground.ChessBoardEditorSettings` ?

Reply:
Done, added `BoardPrefs.toBoardSettings()`

(also rebased to fix conflicts)